agent/lib/kontena/workers/node_stats_worker.rb

```diff
@@ -116,7 +116,7 @@ def publish_node_stats
         network: network_traffic,
         time: Time.now.utc.to_s
       }
-      rpc_client.async.notification('/nodes/stats', [data])
+      rpc_client.async.notification('/nodes/stats', [data]) if rpc_client.connected?
```

Comment:
How much effect does the `... if rpc_client.connected?` have? The `RpcClient#notification` -> `WebsocketClient#send_message` already skips sending if the `@ws` is nil, and AFAICT that ends up being exactly the same thing as `rpc_client.connected?`... so this just introduces an extra sync actor call of overhead?